MovieDB/MovieDB/Services/API/Endpoint/GetMovieDetailInput.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+final class GetMovieDetailInput: Endpoint {
```

Comment:
It will clearer if change the Endpoint to BaseInput.
Because then maybe request Post with more params on Body or Upload file. So, the Endpoint won't meaning enough.